apps/aecontract/test/aect_test_utils.erl

```diff
@@ -303,17 +304,21 @@ contract_filename(Compiler, Name) ->
 
 compile_filename(FileName) ->
     compile(sophia_version(), FileName).
-
```

Comment:
Why remove the empty line? Why do you bundle some functions with a different arity but not the rest? It is not the standard syntax and it is not consistent with the other functions in the file. The same comment stands true for the rest of the changes in this file (functions compile_filename/1,2 and compile/2,3)